tests/framework/ROM/TimeSeries/ARMA/arma_reseed_test.xml

```diff
@@ -1,93 +1,92 @@
+<Simulation verbosity="debug">
+  <TestInfo>
+    <name>framework/ROM/TimeSeries/ARMA.ARMAreseedTest</name>
+    <author>talbpaul</author>
+    <created>2017-07-12</created>
+    <classesTested>SupervisedLearning.ARMA</classesTested>
+    <description>
+      Runs an unpickled ARMA ROM in internal parallel and assures the histories
+      generated are not identical (expected_fail = True).  Note that the gold file
+      is created by coping ARMAparallel/results_1.csv to gold/ARMAparallel/results_0.csv,
+      then we make sure they are not the same through expected_fail
+    </description>
+    <revisions>
       <revision author="talbpaul" date="2017-07-12">created</revision>
       <revision author="alfoa" date="2017-12-13">adapted for new DataObject structure</revision>
-      <revision author="alfoa" date="2018-04-26">Modified location of the test (ARMA folder)</revision>
-    </revisions>
-  </TestInfo>
-
-    <RunInfo>
-      <WorkingDir>ARMAparallel</WorkingDir>
-      <Sequence>read,run,write</Sequence>
-      <batchSize>4</batchSize>
-      <internalParallel>True</internalParallel>
-    </RunInfo>
-
-    <Files>
-        <Input name="pk">arma.pk</Input>
-    </Files>
-
-    <Distributions>
-		  <Uniform name="uniDist">
-			  <lowerBound>0</lowerBound>
-			  <upperBound>1</upperBound>
-		  </Uniform>
-	</Distributions>
-
-	<Samplers>
-		<MonteCarlo name="mc">
-			<samplerInit>
-        <limit>4</limit>
-		    <initialSeed>20021986</initialSeed>
-			</samplerInit>
-      <constant name="scaling">1</constant>
-    </MonteCarlo>
-	</Samplers>
-
-	<Models>
-    <ROM name='ARMA' subType='pickledROM'/>
-  </Models>
-
-  <Steps>
-    <IOStep name='read'>
-      <Input  class='Files'  type = ''    >pk</Input>
-      <Output class='Models' type = 'ROM' >ARMA</Output>
-    </IOStep>
-		<MultiRun name='run'>
-			<Input  class='DataObjects' type='HistorySet'>dataIn</Input>
-			<Model class='Models' 		  type='ROM' 	 		 >ARMA</Model>
-      <Sampler class="Samplers" type="MonteCarlo">mc</Sampler>
-      <Output class="DataObjects" type="HistorySet">results</Output>
-		</MultiRun>
-    <IOStep name='write'>
-			<Input  class='DataObjects' type='HistorySet'>results</Input>
-			<Output class='OutStreams'  type='Print'>results</Output>
-		</IOStep>
-  </Steps>
-
-    <OutStreams>
-        <Print name = 'results'>
-          <type>csv</type>
-          <source>results</source>
-          <what>input, output</what>
-        </Print>
-	</OutStreams>
-
-    <DataObjects>
-      <PointSet name="dataIn">
-        <Input>scaling</Input>
-        <Output>OutputPlaceHolder</Output>
-      </PointSet>
-      <HistorySet name="results">
-        <Input>scaling</Input>
-        <Output>Speed,Time</Output>
-        <options>
-            <pivotParameter>Time</pivotParameter>
-        </options>
-      </HistorySet>
-    </DataObjects>
-</Simulation>
-
-
+      <revision author="alfoa" date="2018-04-26">Modified location of the test (ARMA folder)</revision>
```

Comment:
update the TestInfo regarding this modifications.

Reply:
Any changes here were just due to formatting in the conversion script; I think perhaps Windows line endings were removed.  Nothing was revised about the test, as far as I can tell.